diff--git a/rip.pl b/rip.pl
old mode 100644
new mode 100755
index 59dfcbe..ecc4f14
--- a/rip.pl
+++ b/rip.pl
@@ -1,4 +1,4 @@
-#! c:\perl\bin\perl.exe
+#!/usr/bin/perl
 #-------------------------------------------------------------------------
 # Rip - RegRipper, CLI version
 # Use this utility to run a plugins file or a single plugin against a Reg
@@ -57,6 +57,8 @@ my %config;
 Getopt::Long::Configure("prefix_pattern=(-|\/)");
 GetOptions(\%config,qw(reg|r=s file|f=s csv|c dirty|d auto|a autoTLN|aT guess|g user|u=s sys|s=s plugin|p=s update|uP list|l help|?|h));
 
+my $plugindir = "/usr/share/regripper/plugins/";
+
 # Retrieves absolute path to executable 
 my $scriptpath = Cwd::realpath($0); 
